utils.js

```diff
@@ -0,0 +1,10 @@
+const { expect } = require('@playwright/test');
+
+async function verifyItemsVisible(page, ...itemNames) {
```

Comment:
I would be very careful with such generous functions, and also in the utils file:
- It's a visibility check for text only.
- If it is used in many places, it will be hard to refactor or extend.
In short, it's too generous for such an important assertion.